doc/api/crypto.md

```diff
@@ -668,6 +668,12 @@ provided, `otherPublicKey` is expected to be a [`Buffer`][], `TypedArray`, or
 If `outputEncoding` is given a string will be returned; otherwise a
 [`Buffer`][] is returned.
 
+* `ecdh.computeSecret` will throw an 
```

Comment:
> added: v0.0.0

We only use `added` for new functions/classes/modules/etc.

What you want is something like this, added to the YAML block for this function:

``` 

changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/16849
    description: The default `computeSecret` option is supported now-
```

Reply:
Oh, wait. I think I misunderstood; can you not make this a list item (not let this start with a `*`)?

And I don't think you need a changelog entry if all that changed is the error format.

Reply:
what version should be instead of replace me?

Reply:
forget it, i saw its 10 :)